build/karma.conf.js

```diff
@@ -55,7 +55,7 @@ module.exports = function(config) {
 
     logLevel: 'WARN',
 
-    frameworks: ['jasmine', 'browserify'],
+    frameworks: ['jasmine-jquery', 'jasmine', 'browserify'],
```

Comment:
Using `$('foo')` is still not possible. I guess angular internally uses it now but we can't do this directly.